### PR TITLE
Remove audit vouchers and update README

### PR DESCRIPTION
**Pull request by joshuahannan**
Audit vouchers aren't used any more so this removes everything related to them from the repo

Also adds import addresses to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,7 +156,12 @@ if they have a brand new account, or have been staking through the locked accoun
 
 ### Is the staking collection mandatory
 
-The Flow team will be using the staking collection for Flow Port by default and we recommend that other services use it instead of any of the other account setups. If you provide a staking service for ledger or blocto users, you will need to upgrade to this if you want to give your users access to the entire functionality of their account if they are also using Flow Port. If their entire interaction with staking is through Flow Port, then all the changes are handled for them and there is nothing for you to worry about.
+Flow Port uses staking collection transaction by default other services are encouraged
+to use it instead of any of the other account staking setups.
+If you provide a staking service for ledger or blocto users, you will need to upgrade
```
Comment (franklywatson) [inline README.md:161]:
```suggestion
If you provide a staking service for [Ledger](https://www.ledger.com/) or [Blocto](https://blocto.io/download) users, you will need to upgrade
```

**Review by franklywatson — APPROVED** [review]

**Review by sisyphusSmiling — COMMENTED** [review]
Changes look good, but Go tests seem to be failing

**Comment by joshuahannan** [comment]
@janezpodhostnik I removed the Audit Vouchers contract in this PR because we don't use it anymore, but it looks like flow-go is still using it in its bootstrapping. Are you okay with removing that from the bootstrapping so we can remove it from here?

**Comment by janezpodhostnik** [comment]
@joshuahannan thank you for removing it!

Feel free to merge this, I will resolve the breaking changes in the FVM. 

I will track progress under this issue: https://github.com/onflow/flow-go/issues/2635.

**Comment by janezpodhostnik** [comment]
Can we really remove the contract from mainnet/testnet? 
Should we update the deployed contracts to put a comment on them that they are obsolete and no longer used?

**Comment by joshuahannan** [comment]
We can't remove the contracts from mainnet or testnet, but since they are obsolete, I'm removing them here. Updating them with the comment is a good idea.

@nvdtf Do you know where the information for the Audit Vouchers contract account is so we can update it?

**Comment by nvdtf** [comment]
The contract is deployed to the service account on each network.

**Pull request merged**